### PR TITLE
contentful <3 gatsby v3

### PR DESCRIPTION
**Pull request by axe312ger**
A number of small code style / refactoring changes to get rid of a bunch of warnings (from deprecations) in Gatsby v3

### CODE REVIEW

File: packages/gatsby-source-contentful/src/gatsby-node.js [hunk 1]
```diff
@@ -247,10 +247,10 @@ exports.sourceNodes = async (
         if (node.internal.owner !== `gatsby-source-contentful`) {
           return
         }
-        touchNode({ nodeId: node.id })
+        touchNode(node)
         if (node.localFile___NODE) {
           // Prevent GraphQL type inference from crashing on this property
-          touchNode({ nodeId: node.localFile___NODE })
+          touchNode({ ...node, id: node.localFile___NODE })
```
Comment (wardpeet) [inline packages/gatsby-source-contentful/src/gatsby-node.js:253]: Could you update to use the getNode syntax
```suggestion
          touchNode(getNode(node.localFile___NODE )
```

**Review by wardpeet — COMMENTED** [review]
One comment

**Review by ascorbic — APPROVED** [review]

**Comment by vladar** [comment]
Published in `gatsby-source-contentful@5.0.3`

**Comment by bnbabu55** [comment]
> Published in `gatsby-source-contentful@5.0.3`

@vladar I am facing the same issue with gatsby-source-wordpress plugin 5.0.0, any help would be appreciated.

**Pull request merged**